File: src/main/java/com/graqr/threshr/model/redsky/products/EnvironmentalSegmentation.java
package com.graqr.threshr.model.redsky.products;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record EnvironmentalSegmentation(
        @JsonProperty("is_hazardous_material")
        Boolean isHazardousMaterial) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/EnvironmentalSegmentation.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class EnvironmentalSegmentation(
    @param:JsonProperty("is_hazardous_material") val isHazardousMaterial: Boolean
) {}
